geniza/corpus/templates/corpus/snippets/document_tabs.html

```diff
@@ -4,13 +4,21 @@
     <ul class="tabs">
         <li><a href="{% url 'corpus:document' pk=document.pk %}">{% translate "Document Details" %}</a></li>
         {% with n_records=document.footnotes.count %}
-            {# Translators: n_records is number of scholarship records #}
-            {% blocktranslate asvar srec_text %}Scholarship Records ({{ n_records }}){% endblocktranslate %}
+            {% if n_records == 1 %}
+                {% blocktranslate asvar srec_text %}1 Scholarship Record{% endblocktranslate %}
+            {% else %}
+                {# Translators: n_records is number of scholarship records #}
+                {% blocktranslate asvar srec_text %}{{ n_records }} Scholarship Records{% endblocktranslate %}
+            {% endif %}
             <li>{% if n_records > 0 %}<a href="{% url 'corpus:document-scholarship' pk=document.pk %}">{{ srec_text }}</a>{% else %}<span>{{ srec_text }}</span>{% endif %}</li>
         {% endwith %}
         {% with n_links=document.fragment_urls|length %}
-            {# Translators: n_links is number of external links #}
-            {% blocktranslate asvar elink_text %}External Links ({{ n_links }}){% endblocktranslate %}
+            {% if n_links == 1 %}
+                {% blocktranslate asvar elink_text %}1 External Link{% endblocktranslate %}
+            {% else %}
+                {# Translators: n_links is number of external links #}
+                {% blocktranslate asvar elink_text %}{{ n_links }} External Links{% endblocktranslate %}
+            {% endif %}
```

Comment:
Gissoo and I discussed this change and decided it made more sense the way it was and the tabs don't need to match the search result display (since here they are page labels, where on the search result they are describing the document). Sorry for that, please revert the change.

For future reference, here's an example of a block translate that needs to handle plural and singular; you have to pass a counter https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#blocktranslate-template-tag